### PR TITLE
Add Badges

### PR DESCRIPTION
**Pull request by donavanbecker**
Add Badges for NPM and GitHub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 
 # Homebridge
+[![Website](https://img.shields.io/website?down_color=red&down_message=offline&up_color=green&up_message=online&url=https%3A%2F%2Fhomebridge.io)](https://homebridge.io)
```
Comment (alexindigo) [inline README.md]: Do we need "website: online"  badge?

File: README.md [hunk 2]
```diff
@@ -4,6 +4,11 @@
...
+[![npm](https://img.shields.io/npm/v/homebridge?style=flat)](https://www.npmjs.com/package/homebridge)
+[![npm](https://img.shields.io/npm/dt/homebridge?style=flat)](https://www.npmjs.com/package/homebridge)
+[![GitHub issues](https://img.shields.io/github/issues/nfarina/homebridge?style=flat)](https://github.com/nfarina/homebridge/issues)
```
Comment (alexindigo) [inline README.md]: And number of open issues, not sure what's the benefit to have it a a badge. Same for PRs.

File: README.md (hunk 1 above)
Comment (donavanbecker) [reply README.md]: not necessarily, but gives it a link to it? I can take it off?

File: README.md (hunk 2 above)
Comment (donavanbecker) [reply README.md]: removed it.

File: README.md (hunk 1 above)
Comment (alexindigo) [reply README.md]: We can link to the website from the logo, it's big and prominent and can use badges to other things, like link to the community chat, which is a common practice among many projects.

Something like this:

```
[![Slack Status](https://homebridge-slackin.glitch.me/badge.svg)](https://homebridge-slackin.glitch.me)
```

Which already exists on the page, but might make sense to bring it up top, along with the discord badge:

```
[![Discord Status](https://img.shields.io/discord/512378789297258528.svg?style=flat)](https://discord.gg/RcV7fa8)
```

Comment (donavanbecker) [reply README.md]: Sounds good to me!

**Comment by alexindigo** [comment]
After the last update logo is broken:

Broken logo

PS. And you can put badges on the same line with the header, would look more streamlined.

Thank you.

**Comment by donavanbecker** [comment]
Yeah trying to get image to go to website

**Comment by donavanbecker** [comment]
> After the last update logo is broken:
> 
> Broken logo
> 
> PS. And you can put badges on the same line with the header, would look more streamlined.
> 
> Thank you.

Okay everything is fixed let me know what you think @alexindigo

**Comment by alexindigo** [comment]
Preview

LGTM @nfarina

**Comment by nfarina** [comment]
Looks great, thanks!

**Pull request merged**